.gitignore

```diff
@@ -29,4 +29,5 @@ src/tests/locations.json
 cypress/screenshots
 cypress/videos
 graphql.schema.json
-*.tar.gz
\ No newline at end of file
+*.tar.gz
+9. config: Translations
```

Comment:
The gitignore doesn't seem to work here :/ maybe `*.xlsx` would suffice?

Reply:
Yeah will do.  This was also because I removed the extension in the filename that actually goes on Drive.